src/scripts/process_us_ca.py

```
@@ -0,0 +1,198 @@
+
+def main():
+    final_data = []
+    print("법률 데이터 정제 및 수집 V2.0")
+
+    # 1. 미국 데이터 (pile-of-law / uscode)
+    print("\n🇺🇸 [US] 수집 중...")
+    try:
+        us_ds = load_dataset("pile-of-law/pile-of-law", "uscode", split="train", streaming=True, trust_remote_code=True)
+        count = 0
+        for item in us_ds:
+            if count >= TARGET_PER_COUNTRY: break
```

Ruff (0.14.7)

Multiple statements on one line (colon)

(E701)